packages/blade/src/components/Radio/Radio.tsx

```diff
@@ -57,10 +56,10 @@ const _Radio: React.ForwardRefRenderFunction<BladeElementRef, RadioProps> = (
 
   if (__DEV__) {
     if (!isInsideGroup) {
-      throwBladeError({
-        moduleName: 'Radio',
-        message: 'Cannot use <Radio /> outside of <RadioGroup />',
-      });
+      // throwBladeError({
```

Comment:
delete this

Reply:
Yeah don't need to delete this check actually, since we could achieve the usecase while being inside RadioGroup only.